website-components-playground/src/pages/tabular-modal-page.js

```diff
@@ -50,105 +41,134 @@ const exampleCustomTitleRow = (
   </Spacings.Inline>
 );
 
-const TabularModalPageExample = (props) => (
-  <LayoutApp>
-    <PlaygroundController
-      // eslint-disable-next-line react/prop-types
-      {...props.pageContext}
-      knobs={[
-        {
-          kind: 'text',
-          name: 'title',
-          label: 'Title',
-          initialValue:
-            'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-        },
-        {
-          kind: 'text-multi',
-          name: 'content',
-          label: 'Content',
-          initialValue: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur nec turpis in risus elementum fringilla. Vestibulum nec vulputate metus, fringilla luctus nisl. Vestibulum mattis ultricies augue sagittis vestibulum. Nulla facilisi. Quisque tempor pulvinar efficitur. Praesent interdum ultrices leo. Vivamus non ex maximus justo egestas suscipit eget sed purus. Aliquam ut venenatis nulla. Fusce ac ligula viverra, blandit augue eget, congue turpis. Curabitur a sagittis leo. Nunc sed quam dictum, placerat nunc quis, luctus erat.\nNam id orci ut risus accumsan pellentesque. Quisque efficitur eu arcu ut tristique. Praesent ornare varius leo, ut consequat lacus rutrum vel. Donec mollis leo id lectus vehicula tempor. Nulla facilisi. Fusce fringilla tellus ac ligula consequat suscipit. Sed consectetur molestie quam eu pulvinar. Interdum et malesuada fames ac ante ipsum primis in faucibus. In hac habitasse platea dictumst.`,
-        },
-        {
-          kind: 'select',
-          name: 'useCustomTitleRow',
-          label: 'Title Row',
-          valueOptions: [
-            { value: 'default', label: 'Default' },
-            { value: 'custom', label: 'Custom (form example)' },
-          ],
-          initialValue: 'default',
-        },
-        {
-          kind: 'select',
-          name: 'hideControls',
-          label: 'Hide Controls?',
-          valueOptions: [
-            { value: false, label: 'No' },
-            { value: true, label: 'Yes' },
-          ],
-          initialValue: true,
-        },
-      ]}
-    >
-      {({ values }) => (
-        <ModalController
-          title={values.title}
-          buttonLabel="Open Tabular Modal Page"
-          containerId={containerId}
+const isBrowser = typeof window !== 'undefined';
+const Router = isBrowser ? BrowserRouter : StaticRouter;
```

Comment:
Is this is necessary for SSR? If so, maybe you can leave a comment explaining the reason.